### ISSUE TITLE
scp error

### ISSUE THREAD

**Issue by raphi12** [darkoperator/Posh-SSH#274]
getting this error when downloading ~160gb from a webspace for a backup with the get-scpfolder command as seen below in the image.

thx for ur help :)

![2019-05-16 09_16_02-Fotos](https://user-images.githubusercontent.com/50694246/57833879-562ead00-77bb-11e9-9cd3-c4cfc023396d.png)

**Comment by darkoperator** [comment]
There is no Get-SCPFolder, are you referring to SFTP?

**Comment by raphi12** [comment]
Sure there is.
Go to code -> docs -> get-scpfolder.md

**Comment by darkoperator** [comment]
You are correct, sorry replied while driving and not enough coffee. What is the target server? ubuntu, centos, also what version of OpenSSH on the target. Can you test with -noprogress option to suppress the progress prompt. Wondering if the issue is there

**Comment by raphi12** [comment]
$OpenBSD: sshd_config,v 1.100 2016/08/15 12:32:04 naddy Exp $
Operating System: Debian GNU/Linux 9 (stretch)
Kernel: Linux 2.6.32-042stab117.14

going to test -noprogress now

**Comment by raphi12** [comment]
still receive the same error with -noprogress

**Comment by darkoperator** [comment]
have not forgotten about the issue. Been trying to replicate it with no luck
